test/libsolidity/syntaxTests/immutable/runtimeCode.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.6.6;
```

Comment:
No version pragmas in syntax tests!

Reply:
I don't know who put that there! I bet it was @christianparpart